internal/backend/sessions/sessioncalls/call.go

```diff
@@ -100,22 +113,12 @@ func (cs *calls) executeCall(ctx context.Context, call sdktypes.SessionCallSpec,
 
 	sl = sl.With(zap.Any("f", callv))
 
-	sl.Debug("calling")
+	sl.Debug("invoking")
```

Comment:
This is a clear sign that you shouldn't have passed a sugared logger in the first place.

Reply:
i use the sugared logger  in this module everywhere else, not worth desugaring it just for this. adapted to sugared logger elsewhere.